site/content/en/docs/overview/_index.md

```diff
@@ -22,6 +22,19 @@ Kueue APIs allow you to express:
 
 A core design principle for Kueue is to avoid duplicating mature functionality in Kubernetes components and well-established third-party controllers. Autoscaling, pod-to-node scheduling and job lifecycle management are the responsibility of cluster-autoscaler, kube-scheduler and kube-controller-manager, respectively. Advanced admission control can be delegated to controllers such as gatekeeper.
 
+## Features overview
+
+- **Job management:** support job queueing based on priorities with different strategies: `StrictFIFO` and `BestEffortFIFO`
```

Comment:
Let's add links to the relevant documentation where possible.
For example, here you can link to `docs/concepts/cluster_queue/#queueing-strategy`